core/domain/collection_domain_test.py

```diff
@@ -641,9 +619,33 @@ class SchemaMigrationUnitTests(test_utils.GenericTestBase):
 tags: []
 title: A title
 """)
+    YAML_CONTENT_V3 = ("""category: A category
+language_code: en
+nodes:
+- acquired_skills:
+  - 0
+  - 1
+  exploration_id: Exp1
+  prerequisite_skills: []
+objective: ''
+schema_version: 2
+skills:
+  0:
```

Comment:
OK, but we need to be quite careful to make sure it's clear they're strings throughout and never cast to number. I considered maybe calling them 's0', 's1', etc. -- maybe that might be better?